networkx/algorithms/broadcasting.py

```diff
@@ -96,31 +111,25 @@ def tree_broadcast_center(G):
 def tree_broadcast_time(G, node=None):
     """Return the Broadcast Time of the tree G.
 
-    Broadcasting is an information dissemination problem in which one vertex in a graph, called the originator,
-    must distribute a message to all other vertices by placing a series of calls along the edges of the graph.
-    Once informed, other vertices aid the originator in distributing the message.
-    The broadcasting must be completed as quickly as possible subject to the following constraints:
-    - Each call requires one unit of time.
-    - A vertex can only participate in one call per unit of time.
-    - Each call only involves two adjacent vertices: a sender and a receiver.
-    The minimum broadcast time of a vertex is defined as the minimum amount of time required to complete
-    broadcasting starting from the originator.
-    The broadcast time of a graph is the maximum over all nodes of the
-    minimum broadcast time from that node [1]_.
+    The minimum broadcast time of a node is defined as the minimum amount
+    of time required to complete broadcasting starting from the
+    originator. The broadcast time of a graph is the maximum over
+    all nodes of the minimum broadcast time from that node [1]_.
```

Comment:
I think this paragraph should finish with what the function returns for different inputs. Something like this could work -- but you know best.
```suggestion
    all nodes of the minimum broadcast time from that node [1]_.
    This function returns the minimum broadcast time of `node`.
    If `node` is None the broadcast time for the graph is returned. 
```